src/main/java/Duke/Task/Deadline.java

```diff
@@ -0,0 +1,55 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Deadline-type Task is a task consisting of its details and deadline in day.
+ */
+public class Deadline extends Task {
+    private static final DateTimeFormatter inputDateFormat = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+    private static final DateTimeFormatter outputDateFormat = DateTimeFormatter.ofPattern("MMM d yyyy");
+    /**
+     * The deadline in day.
+     */
+    protected LocalDate by;
```

Comment:
```suggestion
    protected LocalDate deadlineDate;
```
I was a little confused by what `by` referred to until I read the rest of your code. Maybe calling it `deadlineDate` would make the purpose of this variable clearer?